src/makeRobot.js

```diff
@@ -38,7 +38,69 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${robot.coords.x}, y=${robot.coords.y}`;
+    },
+
+    goForward(step = 0) {
+      if (!step) {
+        robot.coords.y += 1;
+      } else if (step > 0) {
+        robot.coords.y += step;
+      }
+
+      return this;
+    },
```

Comment:
```suggestion
    goForward(step = 1) {
      if (step > 0) {
        robot.coords.y += step;
      }

      return this;
    },
```
rewrite in all methods